src/ontology/mondo-edit.obo

```diff
@@ -542679,6 +542690,35 @@ relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/
 property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0001-5208-3432
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7717" xsd:anyURI
 
+[Term]
+id: MONDO:0100573
+name: combined generalized and focal epilepsy
+def: "Any epilepsy where patients have both generalized and focal seizure types, with interictal and/or ictal EEG findings that accompany both seizure types. Patients with Dravet syndrome and Lennox-Gastaut syndrome may have combined focal and generalized epilepsy." [https://www.epilepsydiagnosis.org/epilepsy/focal-generalized-epilepsy-groupoverview.html]
+is_a: MONDO:0005027 {source="https://www.epilepsydiagnosis.org/epilepsy/focal-generalized-epilepsy-groupoverview.html"} ! epilepsy
+intersection_of: MONDO:0005384 ! focal epilepsy
+intersection_of: MONDO:0100574 ! generalized epilepsy
+property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0001-5208-3432
+property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/8280" xsd:anyURI
+
+[Term]
+id: MONDO:0100574
+name: generalized epilepsy
+def: "Epilepsy that is characterized by generalized seizure types and may have typical interictal and/or ictal EEG findings that accompany generalized seizure types (for example generalized spike-wave)." [https://www.epilepsydiagnosis.org/epilepsy/generalized-epilepsy-groupoverview.html]
+xref: NCIT:C3021 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005027 {source="https://www.epilepsydiagnosis.org/epilepsy/generalized-epilepsy-groupoverview.html"} ! epilepsy
+property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0001-5208-3432
+property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/8280" xsd:anyURI
+
+[Term]
+id: MONDO:0100575
+name: hereditary generalized epilepsy
+def: "An instance of generalized epilepsy that is caused by an inherited genomic modification in an individual. This does not always mean that the epilepsy is inherited or can be transmitted to offspring, as the genetic etiology may be a de novo pathogenic variant, or the genetic etiology may have complex/polygenic inheritance." [https://www.epilepsydiagnosis.org/epilepsy/generalized-epilepsy-groupoverview.html, MONDO:patterns/hereditary]
```

Comment:
@nicolevasilevsky 
"as the genetic etiology may be a de novo pathogenic variant, or the genetic etiology may have complex/polygenic inheritance."
I don't think this is correct. 
"hereditary" means that it is inherited. 


I think we need the following: 

Genetic ~ (ie due to some gene/genomic thing)
- inherited (ie that is inherited)
- child that is polygenic (no need for a grouping term)
- child that is de novo (no need for a grouping term)

Reply:
good points. To confirm, you are suggesting that this is renamed to genetic generalized epilepsy (as is it is called in ILAE). We can use/modify the ILAE def:

A generalized epilepsy that is understood to have a genetic etiology. This does not always mean that the epilepsy is inherited or can be transmitted to offspring, as the genetic etiology may be a de novo pathogenic variant, or the genetic etiology may have complex/polygenic inheritance.

To confirm - should I add the subclasses
'inherited generalized epilepsy'
'polygenic generalized epilepsy'
'de novo generalized epilepsy'

?

Reply:
I added these new terms

Reply:
I don't think that we needed "polygenic" and "de novo" as actual classes, but I think it is ok to have them.